app/src/features/editor/hooks/useCompile.tsx

```diff
@@ -45,7 +45,7 @@ export function useCompile(
 
     // TODO: Determine the URL based on the NODE_ENV.
     const server_uri = 'https://api.sway-playground.org/compile';
-    // const server_uri = 'https://127.0.0.1/compile';
+    // const server_uri = 'http://0.0.0.0:8080/compile';
```

Comment:
guessing this is just for handy debugging purposes?

Reply:
Yep, exactly